gdal/ogr/ogrsf_frmts/sqlite/ogrsqliteutility.cpp

```diff
@@ -41,6 +41,28 @@
 
 CPL_CVSID("$Id$")
 
+
+SQLResult::SQLResult ()
+    : papszResult(nullptr), nRowCount(0), nColCount(0), pszErrMsg(nullptr), rc(0) {}
```

Comment:
I would use just C++11 inline member variable declaration. That  makes it more obvious that every member is properly initialized